worlds/openrct2/__init__.py

```diff
@@ -0,0 +1,576 @@
+        eligible_rides = [item for index, item in enumerate(logic_table) if
+                          item in item_info["rides"] and item not in item_info["non_starters"]]
+        eligible_rides = list(dict.fromkeys(eligible_rides))
+        self.random.shuffle(eligible_rides)
+        if self.options.required_unique_rides.value:
+            count = 0
+            while count < self.options.required_unique_rides.value:
+                self.unique_rides.append(eligible_rides[count])
+                count += 1
+            # self.unique_rides = [eligible_rides[i] for i in
+            #                     eligible_rides[-self.multiworld.required_unique_rides[self.player].value:]]
+        # print("Here's the eligible rides:")
+        # print(eligible_rides)
+        # print("Here's what was chosen:")
+        # print(self.unique_rides)
+        for ride in self.unique_rides:
+            add_rule(self.multiworld.get_region("Victory", self.player).entrances[0],
+                     lambda state, selected_prereq=ride: state.has(selected_prereq, self.player))
+
+    def generate_basic(self) -> None:
+        # place "Victory" at the end of the unlock tree and set collection as win condition
+        self.multiworld.get_location("Victory", self.player).place_locked_item(
+            OpenRCT2Item("Victory", ItemClassification.progression, None, self.player))
+        self.multiworld.completion_condition[self.player] = lambda state: state.has("Victory", self.player)
+
+    def write_spoiler(self, spoiler_handle: TextIO) -> None:
+        spoiler_handle.write(f'Starting Ride:       {self.starting_ride}\n')
+
+    def fill_slot_data(self):
+        # archipelago_objectives = {Guests: [300, false], ParkValue: [0, false], RollerCoasters: [5,2,2,2,0,false],
+        # RideIncome: [0, false], ShopIncome: [8000, false], ParkRating: [700, false], LoanPaidOff: [true, false],
+        # Monopoly: [true, false]};
+        guests = self.options.guest_objective.value
+        park_value = self.options.park_value_objective.value
+        roller_coasters = self.options.roller_coaster_objective.value
+        excitement = self.options.roller_coaster_excitement.value
+        intensity = self.options.roller_coaster_intensity.value
+        nausea = self.options.roller_coaster_nausea.value
+        park_rating = self.options.park_rating_objective.value
+        pay_off_loan = self.options.pay_off_loan.value
+        monopoly = self.options.monopoly_mode.value
+        unique_rides = self.unique_rides
+        objectives = {"Guests": [guests, False], "ParkValue": [park_value, False],
+                      "RollerCoasters": [roller_coasters, excitement, intensity, nausea, 0, False],
+                      "RideIncome": [0, False], "ShopIncome": [0, False], "ParkRating": [park_rating, False],
+                      "LoanPaidOff": [pay_off_loan, False], "Monopoly": [monopoly, False],
+                      "UniqueRides": [unique_rides, False]}
```

Comment:
I'm creating the objects here for the actual plugin to handle. When the game receives this object, it will directly copy the tuples to its memory and track the values. Once the correct value is achieved (Such as getting enough guests in the park), it will flip the boolean. Once all the booleans are flipped, the win signal is sent out. As for the previous list in items.py, the reason is simple. Past Colby is bad at programming and apparently thought he'd use that value someday